src/dials/algorithms/scaling/scaling_library.py

```diff
@@ -105,10 +105,8 @@ def choose_initial_scaling_intensities(reflection_table, intensity_choice="profi
             ] * flex.pow2(conv * inverse_partiality)
             if "partiality.inv.variance" in reflection_table:
                 reflection_table["variance"] += (
-                    reflection_table["intensity.sum.value"]
-                    * conv
-                    * reflection_table["partiality.inv.variance"]
-                )
+                    (reflection_table["intensity.sum.value"] * conv) ** 2
```

Comment:
I see that `flex.pow2` is used elsewhere.  Does that make sense here too?
```suggestion
                    flex.pow2(reflection_table["intensity.sum.value"] * conv)
```

Reply:
Agree on `flex.pow2()` because it's idiomatic